pyaqueduct/client/client.py

```diff
@@ -15,6 +15,7 @@
 from tqdm import tqdm
 
 from pyaqueduct.client.types import ExperimentData, ExperimentsInfo, TagsData
+from pyaqueduct.client.plugin_types import PluginData, PluginExecutionResultData
```

Comment:
If you have made this decision to separate the types in different modules, please refactor the experiment types to be collected in `experiment_types` module.